src/lib/wasm/ConstellationStore.ts

```diff
@@ -91,6 +91,25 @@ class ConstellationStore {
         return failure(WarpError.CONSTELLATION_NOT_FOUND)
     }
 
+    /**
+     * moves item/s from constellation.
+     * @returns A Result containing either success or failure with a WarpError.
+     */
+    async dropIntoFolder(fileName: string, toFolderName: string): Promise<Result<WarpError, void>> {
```

Comment:
Just noticed but maybe we could remove the async signature and the promise since we are not awaiting a future (or js promise). Thoughts?

Reply:
I was totally thinking we wouldn't need it when I wrote it but threw it in outta caution, so, definitely agree!